ci/ios.groovy

```diff
@@ -19,6 +19,10 @@ def bundle(type) {
   plutil('CFBundleShortVersionString', cmn.version())
   plutil('CFBundleVersion', cmn.genBuildNumber())
   plutil('CFBundleBuildUrl', currentBuild.absoluteUrl)
+  if (type == 'pr') { /* PR builds shouldn't replace normal releases */
+    env.BUNDLE_ID_SUFFIX = '.pr'
+    //plutil('CFBundleIdentifier', 'im.status.ethereum.pr')
```

Comment:
commented-out line... is that expected?

Reply:
Considering PR has `Status: WIP` in description, yes :D.